feature-libs/customer-ticketing/core/customer-ticketing-constants.ts

```diff
@@ -0,0 +1 @@
+export const MAX_INPUT_CHARACTERS = 2000;
```

Comment:
will this be re-used in other places? If not, keep it within the reopen component as a property

Reply:
Can most likely re-use it for `feature-libs/customer-ticketing/core/config/default-customer-ticketing-config.ts`

maybe make the other set default value as constants here. All that being said, the array as well

Reply:
Yes, It will be re-used in messaging component.